stable/storageos-operator/README.md

```diff
@@ -0,0 +1,199 @@
+
+```yaml
+cluster:
+  ...
+  create: false
+```
+
+Create a secret to store storageos cluster secrets:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "storageos-api"
+  namespace: "default"
+  labels:
+    app: "storageos"
+type: "kubernetes.io/storageos"
+data:
+  # echo -n '<secret>' | base64
+  apiAddress: c3RvcmFnZW9zOjU3MDU=
+  apiUsername: c3RvcmFnZW9z
+  apiPassword: c3RvcmFnZW9z
+```
+
+Create a `StorageOSCluster` custom resource and refer the above secret in
+`secretRefName` and `secretRefNamespace` fields.
+
+```yaml
+apiVersion: "storageos.com/v1"
+kind: "StorageOSCluster"
+metadata:
+  name: "example-storageos"
+  namespace: "default"
+spec:
+  secretRefName: "storageos-api"
+  secretRefNamespace: "default"
+```
+
+Once the `StorageOSCluster` configuration is applied, the StorageOSCluster
+operator will create a StorageOS cluster in the `storageos` namespace by
+default.
+
+Most installations will want to use the default [CSI](https://kubernetes-csi.github.io/docs/)
+driver.  To use the [Native Driver](https://kubernetes.io/docs/concepts/storage/volumes/#storageos)
+instead, disable CSI:
+
+```yaml
+spec:
+  ...
+  csi:
+    enable: false
+  ...
+```
+
+in the above `StorageOSCluster` resource config.
+
+Learn more about advanced configuration options
+[here](https://github.com/storageos/cluster-operator/blob/master/README.md#storageoscluster-resource-configuration).
+
+To check cluster status, run:
+
+```bash
+$ kubectl get storageoscluster
+NAME                READY     STATUS    AGE
+example-storageos   3/3       Running   4m
+```
+
+All the events related to this cluster are logged as part of the cluster object
+and can be viewed by describing the object.
+
+```bash
+$ kubectl describe storageoscluster example-storageos
+Name:         example-storageos
+Namespace:    default
+Labels:       <none>
+...
+...
+Events:
+  Type     Reason         Age              From                       Message
+  ----     ------         ----             ----                       -------
+  Warning  ChangedStatus  1m (x2 over 1m)  storageos-operator  0/3 StorageOS nodes are functional
+  Normal   ChangedStatus  35s              storageos-operator  3/3 StorageOS nodes are functional. Cluster healthy
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the StorageOSCluster
+Operator chart and their default values.
+
+Parameter | Description | Default
+--------- | ----------- | -------
+`operator.image.repository` | StorageOSCluster container image repository | `storageos/cluster-operator`
+`operator.image.tag` | StorageOSCluster container image tag | `1.1.0`
+`operator.image.pullPolicy` | StorageOSCluster container image pull policy | `IfNotPresent`
+`podSecurityPolicy.enabled` | If true, create & use PodSecurityPolicy resources | `false`
+`podSecurityPolicy.annotations` | Specify pod annotations in the pod security policy | `{}`
+`cluster.create` | If true, auto-create the StorageOS cluster | `true`
```

Comment:
if we auto-create, should we still call the cluster `example-storageos`?

Reply:
Where are we calling it that?
`cluster.name` is `storageos`, right below this.